File: Android/_setup/_ProjectName_/src/main/java/com/twist/tmstore/entities/FeeData.java
package com.twist.tmstore.entities;

/**
 * Created by Twist Mobile on 8/3/2016.
 */

public class FeeData {
    public enum Type {
        FIXED("fixed"),
        PERCENT("percent");

        private final String value;

        Type(String value) {
            this.value = value;
        }

        public String getValue() {
            return this.value;
        }

        public static Type from(String name) {
            if (name != null && !name.equals("")) {
                for (FeeData.Type type : values()) {
                    if (type.getValue().equalsIgnoreCase(name)) {
                        return type;
                    }
                }
            }
            return Type.FIXED;
        }
    }

    public String plugin_title;
    public String label;
    public boolean taxable;
    public float minorder;
    public float cost;
    public Type type;
}

File: IOS/TMStore/TMStore/Classes/Models/FeeData.m
//
//  FeeData.m
//  TMStore
//
//  Created by Rishabh Jain on 03/11/16.
//  Copyright © 2016 Twist Mobile. All rights reserved.
//

#import "FeeData.h"
static NSMutableArray* _allFeeData = nil;//ARRAY OF FeeData
@implementation FeeData
- (id)init {
    self = [super init];
    if (self) {
        _plugin_title = @"";
        _label = @"";
        _taxable = false;
        _minorder = 0.0f;
        _cost = 0.0f;
        if (_allFeeData == nil) {
            _allFeeData = [[NSMutableArray alloc] init];
        }
        [_allFeeData addObject:self];
    }
    return self;
}
+ (NSMutableArray*)getAllFeeData {
    return _allFeeData;
}
+ (void)resetFeeData {
    [[FeeData getAllFeeData] removeAllObjects];
}
@end